binding-library/python/setup.py

```diff
@@ -0,0 +1,24 @@
+import setuptools
+from setuptools import setup
+
+with open("README.md") as f:
+    long_description = f.read()
+
+setup(
+    name='azure-functions-kafka-binding',
+    version='1.0.0',
```

Comment:
You can add a patch version here to indicate that you're making changes to the package: `1.0.1`

Reply:
@maiqbal11 : Yes functionality works the same. Tested it.